neo/Prompt/Commands/Bootstrap.py

```diff
@@ -92,6 +99,7 @@ def do_bootstrap(bootstrap_file, destination_dir, tmp_file_name='./Chains/bootst
         print("cleaning up %s " % tmp_chain_name)
         if os.path.exists(tmp_chain_name):
             shutil.rmtree(tmp_chain_name)
+            os.remove(tmp_file_name)
```

Comment:
We might want to keep the downloaded file, for test bootstrap perhaps? Cc @localhuman